apps/react/src/features/anime/components/AnimeInfinitiveScroll/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { getAnimeList, getNextAnimeList } from '@js-camp/react/store/anime/dispatchers';
+import {
+  selectAmineList,
+  selectIsAnimeLoading,
+  selectNextPage,
+} from '@js-camp/react/store/anime/selectors';
+import { useAppDispatch, useAppSelector } from '@js-camp/react/store/store';
+
+import { FC, memo, useCallback, useEffect, useRef } from 'react';
+import { useSelector } from 'react-redux';
+
+import { LoadingComponent } from '../../../../components';
+
+import { AnimeItem } from '../AnimeItem';
+
+import style from './AnimeInfinitiveScroll.module.css';
+
+const option = {
+  root: null,
+  rootMargin: '20px',
+  threshold: 0.2,
+};
+
+export const AnimeInfinitiveScrollInner: FC = () => {
+  const itemRef = useRef<HTMLDivElement>(null);
+  const isLoading = useAppSelector(selectIsAnimeLoading);
+  const dispatch = useAppDispatch();
+  const nextPage = useAppSelector(selectNextPage);
+  const animeList = useSelector(selectAmineList);
+
+  useEffect(() => {
+    dispatch(getAnimeList(''));
+  }, []);
+
+  const handleObserver = useCallback(
+    (entries: IntersectionObserverEntry[]) => {
+      const target = entries[0];
+      if (target.isIntersecting && nextPage) {
+        dispatch(getNextAnimeList(nextPage));
+      }
+    },
+    [nextPage],
+  );
+
+  useEffect(() => {
+
+    const observer = new IntersectionObserver(handleObserver, option);
+    if (itemRef.current) {
+      observer.observe(itemRef.current);
+    }
+    return () => {
+      observer.disconnect();
+    };
+  }, [handleObserver]);
+  if (isLoading) {
```

Comment:
Please, pay more attention to code formatting. Use empty lines to group your code into code blocks.
```suggestion
  useEffect(() => {
    const observer = new IntersectionObserver(handleObserver, option);
    if (itemRef.current) {
      observer.observe(itemRef.current);
    }
    return () => {
      observer.disconnect();
    };
  }, [handleObserver]);
 
  if (isLoading) {
```